satellite.py

```diff
@@ -34,86 +34,140 @@
 
 """
 import argparse
+import math
 import dimod
 import itertools
 import json
 import neal
 import sys
 from dwave.system import LeapHybridSampler
+import matplotlib.pyplot as plt
```

Comment:
According to PEP8, [imports should be grouped](https://www.python.org/dev/peps/pep-0008/#imports), for example:
```suggestion
import argparse
import itertools
import json
import math
import sys

import matplotlib.pyplot as plt
import dimod
import neal
from dwave.system import LeapHybridSampler
```